book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
+
```

Comment:
- [ ] Based on the project requirements, you are to have a getter and setter method for rental attribute/data. KIndly implement it. `attr_accessor :rental`. 👍